configuration/concepts/1000_adult_hiv_initial_form.csv

```diff
@@ -223,25 +223,25 @@ a048b263-c12e-4e48-b1eb-1f9a7d11153e,,Normal;Abnormal,Examen digestif,Gastrointe
 0d0ee4db-0b26-48ab-bc64-2364321bd0be,,,Description du diagnostic,Diagnosis description,Description du diagnostic,Diagnosis description,Misc,Text,,
 6913c5a7-c91c-4995-b8ee-a3e87bee7bcd,,,Diagnostic differentiel,Differential diagnosis,Diagnostic differentiel,Differential diagnosis,Misc,Text,,
 9f40f1a4-ec71-4f52-948b-f0b362640278,,,I,I,I,I,Misc,N/A,,
 4d6b53e5-322c-462e-a35c-42800c3c7fb4,,,II,II,II,II,Misc,N/A,,
 0b53a94c-7700-4381-8f16-f74a09b833cd,,,III,III,III,III,Misc,N/A,,
 fd27fdb0-484a-488c-93b5-15fd894ba4b9,,,IV,IV,IV,IV,Misc,N/A,,
 c06cd011-b37a-4159-9bdd-6f339069bb91,,I;II;III;IV,Stade OMS (entourer la stade correspondant ),WHO stage (circling the corresponding stage),Stade OMS (entourer la stade correspondant ),WHO stage (circling the corresponding stage) (3766),Misc,Coded,,
-d048b999-a723-40af-a0a2-935d6e61c35c,,,"Indiquez l'èlèment classant au stade OMS (par exemple: si : Stade I, prèciser (Asymptomatique è ou Lymphadènopathie gènèralisèe persistante è)","Indicate the classifying element at WHO stage (for example: if: Stage I, specify (Asymptomatic? Persistent generalized lymphadenopathy?) ","Indiquez l'èlèment classant au stade OMS (par exemple: si : Stade I, prèciser (Asymptomatique è ou Lymphadènopathie gènèralisèe persistante è)","Indicate the classifying element at WHO stage (for example: if: Stage I, specify (Asymptomatic? Persistent generalized lymphadenopathy?)",Misc,Text,,
-b44ed79b-ee23-4e98-ada8-4dd4d95ef74c,"Skin ;Ganglionic Area ;Chest Exam ;Exam cardiovascular ;Gastrointestinal examination ;Urogenital examination ;Neurological examination ;Psychiatric examination ;TB screened?;TB screened date;Signs or symptoms suggestive of Tuberculosis;Diagnosis description;Differential diagnosis;WHO stage (circling the corresponding stage);Indicate the classifying element at WHO stage (for example: if: Stage I, specify (Asymptomatic? Persistent generalized lymphadenopathy?)",,3.2 Examen Systèmique,3.2 Systemic Exam,3.2 Examen Systèmique,3.2 Systemic Exam,Misc,N/A,,
-234fb6cd-2998-4c2e-ae9e-63acc0745a98,,Yes full name;No full name,Prescription ARV prescrit,Prescription prescribed ARV,Prescription ARV prescrit,Prescription prescribed ARV,Misc,Coded,,
+d048b999-a723-40af-a0a2-935d6e61c35c,,,"Not ApplicaIndiquez l'élément classant au stade OMS (par exemple : si : Stade I, préciser (Asymptomatique ou Lymphadènopathie généralisée persistante)ble","Indicate the classifying element at WHO stage (for example: if: Stage I, specify (Asymptomatic? Persistent generalized lymphadenopathy?) ","Not ApplicaIndiquez l'élément classant au stade OMS (par exemple : si : Stade I, préciser (Asymptomatique ou Lymphadènopathie généralisée persistante)ble","Indicate the classifying element at WHO stage (for example: if: Stage I, specify (Asymptomatic? Persistent generalized lymphadenopathy?)",Misc,Text,,
```

Comment:
@lauravignoli Looks like a copy/paste typos: see `Not Applica` in the 2 french translations: `Not ApplicaIndiquez l'élément classant au stade OMS ...`

Reply:
Thanks, will remove.